internal/runners/export/config.go

```diff
@@ -0,0 +1,96 @@
+package export
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+)
+
+// Filter is the --filter flag for the export config command, it implements captain.FlagMarshaler
+type Filter int
+
+const (
+	Unknown Filter = iota
```

Comment:
You don't seem to be using Unknown for anything that makes it necessary. ie. all logic relying on it could easily rely on the absence of it instead.

Reply:
I've updated it's name to `unset` and am now using it when creating the params

Reply:
Nvm it can be removed 👍